cmd/gaia/app/genesis.go

```diff
@@ -20,6 +22,8 @@ var (
 	// bonded tokens given to genesis validators/accounts
 	freeFermionVal  = int64(100)
 	freeFermionsAcc = int64(50)
+
+	defaultKeyPass = "1234567890"
```

Comment:
can you change this to "12345678". Since your adding a flag to the CLI here, it may be better to just make this default parameter so it appears in help. 

While you're at it, would you like to just add this somewhere in the docs? (No need for me to make a separate PR for that)

Reply:
Sure thing!